packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -252,25 +305,43 @@ export default function MainFeedLayout({
         user,
         ...Object.values(variables ?? {}),
       ),
-      query: query.query,
+      query: config.query,
       variables,
-      emptyScreen: <FeedEmptyScreen />,
-      header: null,
-      actionButtons: feedWithActions && (
+      header: !isLaptop && (isExplorePopular || isExploreLatest) && (
```

Comment:
Should this not be under the actionButtons already?

Reply:
We can actually do it there, I don't have any strong preference on where to put this. One too many options 😂

Reply:
Ah if we can uniform it in one way of the other I'm fine with that :)

Reply:
Yep. Updated it to use the same property like suggested to make it more consistent.